### PR TITLE
add section about system upgrades in OI hipster

### PR DESCRIPTION
**Pull request by bbarker**

### CODE REVIEW

**Review by JMadgwick — DISMISSED** [review]

File: docs/handbook/systems-administration.md [hunk 1]
```diff
@@ -751,6 +751,23 @@ onto the system before changing the run-level.
...
+
+## Updating All Packages (Upgrading Your System)
+
+OpenIndiana Hipster is a rolling release
+distribution, so to upgrade your system, you simply run `pkg update`
+all packages will be updated to the latest revision:
+
+```sh
+sudo pkg update
+```
```
Comment (AndWac) [inline docs/handbook/systems-administration.md:763]: The canonical way of doing this on Solarish systems is
pfexec pkg update

Comment (bbarker) [reply docs/handbook/systems-administration.md:763]: I was wondering about that. I'll try to do a future update and maybe tag you as a reviewer, and add a section on the differences between the two (without going into a lot of detail on `pfexec` - just sort of a landing entry for people more familiar with `sudo` - I'm quite rusty on `pfexec`, and wasn't sure if things had changed).

Comment (bbarker) [reply docs/handbook/systems-administration.md:763]: So one existing stumbling block, which is interesting:



```$ pfexec pkg update

pkg update: Insufficient access to complete the requested operation.
Please try the operation again as a privileged user.
```

Somehow sudo seems to bypass the privilege mechanisms pfexec is using.

Comment (AndWac) [reply docs/handbook/systems-administration.md:763]: Solaris and its derivates have fine grain rights and privilege control, look at my user entry in /etc/user_attr:
andreas::::type=normal;defaultpriv=basic,dtrace_user,file_dac_read,file_dac_write,proc_clock_highres;profiles=Software Installation,CPU Power Management,Service Management,ZFS File System Management

In this case you'll need the profiles=Software Installation part to be able to use pfexec pkg.

Comment (bbarker) [reply docs/handbook/systems-administration.md:763]: Thanks - my last memory of using OI or Solaris, almost a decade ago, was that `sudo` didn't even exist in the installation of the OS, though maybe I'm remembering incorrectly. Was a decision made to include it for compatibility?

Comment (AndWac) [reply docs/handbook/systems-administration.md:763]: You can configure sudo to also don't need a password. You have even configure it to allow passwordless for limited paths. The mechanisms in Solaris evolved over several versions. IIRC roles have been introduced in Solaris 8 while privileges came with Solaris 10.

Comment (AndWac) [reply docs/handbook/systems-administration.md:763]: You might also want to add that one need to have the Software Installation profile in /etc/user_attr for this to work.

Comment (bbarker) [reply docs/handbook/systems-administration.md:763]: Sounds good - I added a variant that doesn't require the user to manually edit `/etc/user_attr`

File: docs/handbook/getting-started.md [hunk 2]
```diff
@@ -1417,7 +1417,45 @@ Use the following steps to change the root password:
     These pages are only available by running the man page viewer locally on your system.
 </div>
 
-### Introduction
+### Quick Start to IPS
+
+As a dekstop user, you will likely want to enable the `Software Installation` profile for
+your user; you can use `sudo` instead of `pfexec` if you are just getting started with
+OpenIndiana and have not yet familiar with roles
+(replace `username` with your actual username):
```
Comment (AndWac) [inline docs/handbook/getting-started.md:1425]: Software Installation is a profile (pfexec = profiled execute, other profiled commands are available, too) while roles are a different thing (if you have been granted a role you can su into it; so roles are special user accounts that cannot login directly). Privileges are the third concept which are low level rights ("fine grain control over the actions of processes" - see man privileges).

Comment (bbarker) [reply docs/handbook/getting-started.md:1425]: Got it - I was using `roles` as a broad term but I likely should have said "illumos RBAC". Is that an acceptable change, or would you like me to integrate your comment into the same paragraph?

File: docs/handbook/getting-started.md [hunk 3]
```diff
@@ -1417,7 +1417,45 @@ Use the following steps to change the root password:
     These pages are only available by running the man page viewer locally on your system.
 </div>
 
-### Introduction
+### Quick Start to IPS
+
+As a dekstop user, you will likely want to enable the `Software Installation` profile for
```
Comment (DrLou) [inline docs/handbook/getting-started.md:1422]: 'desktop' user...

Comment (bbarker) [reply docs/handbook/getting-started.md]: Thanks, I enabled grammarly so I'll hopefully have fewer typos.

**Review by AndWac — APPROVED** [review]

File: docs/handbook/systems-administration.md [hunk 4]
```diff
@@ -751,6 +751,26 @@ onto the system before changing the run-level.
...
+
+OpenIndiana Hipster is a rolling release
+distribution, so to upgrade your system, you simply run `pkg update`
+all packages will be updated to the latest revision:
+
+```sh
+pfexec pkg update
+```
+
+(Note: for `pfexec` to succeed you'll need to have the `Software Installation` profile <a href="../getting-started/#quick-start-to-ips">enabled for your user account</a>.)
```
Comment (JMadgwick) [inline docs/handbook/systems-administration.md:765]: Markdown links are better than using HTML.
```suggestion
(Note: for `pfexec` to succeed you'll need to have the `Software Installation` profile [enabled for your user account](../getting-started/#quick-start-to-ips).)
```

**Comment by bbarker** [comment]
I think all changes are in - probably best to squash. But let me know if anything else is needed for this.

**Review by JMadgwick — APPROVED** [review]

**Pull request merged**